app/controllers/application_controller.rb

```diff
@@ -27,4 +29,14 @@ def error(status_code, exception = nil)
   def set_expiry(age = 60.minutes)
     expires_in age, public: true unless Rails.env.development?
   end
+
+  def set_cors_headers
+    headers['Access-Control-Allow-Origin'] = '*'
+    headers['Access-Control-Allow-Methods'] = 'GET'
+    headers['Access-Control-Allow-Headers'] = 'Origin, X-Requested-With, Content-Type, Accept'
```

Comment:
That makes sense, the only part I'm not sure about is that `Content-Type` will be set to `application/json; charset=utf-8` for the JSON responses so the Mozilla dev doc makes it sound like it won't be a simple request. I don't understand their docs.